service-locator-dns/src/test/scala/com/lightbend/dns/locator/ServiceLocatorSpec.scala

```diff
@@ -153,6 +165,28 @@ class ServiceLocatorSpec extends WordSpec with Matchers with BeforeAndAfterAll {
 
       requestor.expectMsg(Addresses(List.empty))
     }
+
+    "Translate SRV records after resolving" in {
+      val dnsProbe = TestProbe()
+      val serviceLocator = system.actorOf(Props(new TestServiceLocator(dnsProbe)))
+
+      val requestor = TestProbe()
+
+      requestor.send(serviceLocator, ServiceLocator.GetAddress("_test-srv-translator._tcp.marathon.mesos"))
+
+      dnsProbe.expectMsg(Dns.Resolve("_test-srv-translator._tcp.marathon.mesos"))
+      val srv = SRVRecord("_test-srv-translator._tcp.marathon.mesos", 3600, 0, 0, 1000, "some-service-host.marathon.mesos")
+      dnsProbe.sender() ! SrvResolved("some-service", List(srv))
+
+      dnsProbe.expectMsgPF() {
+        case Dns.Resolve("some-service-host.marathon.mesos") =>
+      }
+
+      dnsProbe.sender() ! Dns.Resolved("some-service-host.marathon.mesos", List(InetAddress.getByName("127.0.0.1")))
```

Comment:
We can also use `dnsProbe.reply(Dns.Resolved("some-service-host.marathon.mesos", List(InetAddress.getByName("127.0.0.1"))))` here too.